parsec/backend/utils.py

```diff
@@ -54,6 +54,10 @@ async def wrapper(self: T, client_ctx: Ctx, msg: dict[str, object]) -> dict[str,
                 from parsec.api.protocol import InvitedAnyCmdReq
 
                 typed_req = InvitedAnyCmdReq.load(packb(msg))
+            elif client_ctx.TYPE == ClientType.ANONYMOUS:
+                from parsec._parsec import AnonymousAnyCmdReq
+
+                typed_req = AnonymousAnyCmdReq.load(packb(msg))
             else:
                 from parsec.api.protocol import AuthenticatedAnyCmdReq
```

Comment:
Why don't we use `parse._parsec.AuthenticatedAnyCmdReq` ?

Reply:
Same thing as I answered above